### ISSUE TITLE
File interface not working when editing item in O2M relation

### ISSUE THREAD

**Issue by blackfox00** [directus/app#1788]
# Bug Report
When having a O2M relation and that related collection has File input inside, editing of the image doesn't work (interface is not displayed)

## Steps to Reproduce
Create 2 related collections as O2M, add File interface to related collection, create new master collection item and create related collection item and add file, click on created related item table row to edit it. I don't know if it's related but I also have the O2M relation in Translation interface.

## Expected Behavior
When editing related item File interface is displayed

## Actual Behavior
File interface is not displayed, console error is emitted
```
TypeError: Cannot read property 'split' of undefined
    at a.subtitle (input.vue:116)
    at nr.get (vue.runtime.esm.js:4473)
    at nr.evaluate (vue.runtime.esm.js:4578)
    at a.subtitle (vue.runtime.esm.js:4830)
    at a.n (input.vue?dad7:1)
    at a.e._render (vue.runtime.esm.js:3542)
    at a.r (vue.runtime.esm.js:4060)
    at nr.get (vue.runtime.esm.js:4473)
    at new nr (vue.runtime.esm.js:4462)
    at Pn (vue.runtime.esm.js:4067)
```

## Other Context & Screenshots

https://imgur.com/B26rzzc

I am attaching a recording of simulating the issue. I believe the problem is when fetching the data, only image ID is fetched and not the whole object like `?fields=image.*.*,...` as in the case when you edit the collection item directly 

## Technical Details
API 2.2.0, APP 7.6.0
Device: Desktop
OS: Alpine 3.8
Web Server: nginx 1.14.2
PHP Version: 7.2.18
Database:  mysql 5.7.26
Install Method: docker (updated to pull newest versions)

**Comment by blackfox00** [comment]
I updated to the newest version (7.6.1) and the problem still persists

**Comment by rijkvanzanten** [comment]
I just setup a fresh relation including file on the nightly demo, and it seems to work as expected for me. Could you share your exact setup or (if possible) a database dump so I can reproduce your exact environment and see what's up?

You can also send it on [Slack](https://directus.chat) if you'd rather keep it private

**Comment by blackfox00** [comment]
My O2M interace with file is also inside the translation interface, could that be the difference? I think the problem is that the image does not get fetched, only image ID (`?fields=image.*.*` is missing).  I'll try to prepare some db dump with MVE example later on

**Comment by rijkvanzanten** [comment]
Ah that could be the culprit. Having a dump would help a lot  🙂

**Comment by blackfox00** [comment]
I sent you the setup on slack. Can we reopen this?

**Comment by blackfox00** [comment]
This is seems to be fixed in 7.7.0. (probably by o2m rework)

**Comment by benhaynes** [comment]
Thank you for confirming!
